runtimes/instance-rootfs/create-debian-11-disk.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -euf
```

Comment:
Add `-o pipefail` ?

Reply:
It makes sure that any failure in a piped operation (ex: `echo "hello" | cat`) causes the script to stop.

Reply:
Okay, but here don't make sense IMO, because we don't use any pipe command.